app/scripts/views/page.js

```diff
@@ -83,7 +88,7 @@ var Page = React.createClass({
                     <div className='row'>
                           <div className='navbar-header'>
                             <Link className='navbar-brand col-xs-12' to='home'>
-                              <img src='./static/img/logo.png' alt='IPFS' className='img-responsive logo'/><span className='sr-only'>{i18n.t('IPFS')}</span>
+                              <img src={require('../../img/logo.png')} alt='IPFS' className='img-responsive logo'/><span className='sr-only'>{i18n.t('IPFS')}</span>
```

Comment:
> This is much less pretty. Can we do something about ../../?

Yes and no, in the long run we need an image component like this: `<Image src="logo.png" />` that handles these things for us. But for now it's just getting things working with the new setup